R/arrange.R

```diff
@@ -125,47 +125,11 @@ arrange_rows <- function(data,
     quosure
   })
 
-  # give the quosures arbitrary names so that
-  # data has the right number of columns below after transmute()
-  quo_names <- vec_paste0("^^--arrange_quosure_", seq_along(quosures))
-  names(quosures) <- quo_names
-
-  # TODO: not quite that because when the quosure is some expression
-  #       it should be evaluated by groups.
-  #       for now this abuses transmute so that we get just one
-  #       column per quosure
-  #
-  #       revisit when we have something like mutate_one() to
-  #       evaluate one quosure in the data mask
-  data <- withCallingHandlers({
-    transmute(new_data_frame(data), !!!quosures)
-  }, error = function(cnd) {
-
-    if (inherits(cnd, "dplyr:::mutate_error")) {
-      # reverse the name mangling
-      bullets <- gsub("^^--arrange_quosure_", "..", cnd$bullets, fixed = TRUE)
-      # only name bullets that aren't already named
-      names <- names2(bullets)
-      names[names == ""] <- "x"
-      bullets <- set_names(bullets, names)
-
-      # skip the parent as this has reworked the bullets
-      # and this would be confusing to have them
-      parent <- cnd$parent
-    } else {
-      parent <- cnd
-      bullets <- c()
-    }
-
-    bullets <- c(
-      "Problem with the implicit `transmute()` step. ",
-      bullets
-    )
-    abort(bullets, call = error_call, parent = parent)
-
-  })
-
-  directions <- directions[quo_names %in% names(data)]
+  if (length(quosures) > 0) {
+    names(quosures) <- paste0("..", seq_along(quosures))
+  }
```

Comment:
```suggestion
  names(quosures) <- vec_paste0("..", seq_along(quosures))
```

Could use this if you want to

Reply:
Oooh, I was looking for `str_` something and couldn't find it.